packages/app-project/src/screens/ClassifyPage/ClassifyPage.js

```diff
@@ -39,8 +39,8 @@ function ClassifyPage({
     subjectSetFromUrl = workflowFromUrl.subjectSets.find(subjectSet => subjectSet.id === subjectSetID)
   }
 
-  // The classifier requires a workflow by default
-  let canClassify = !!workflowFromUrl
+  // The classifier requires a workflow ID by default
```

Comment:
Good catch. Maybe the other comments should be specific that the classifier requires IDs too?

Reply:
Yes! Updated accordingly